runtime/src/op/raw/perf/convert/event_scope.rs

```diff
@@ -0,0 +1,21 @@
+use crate::op::raw::perf::convert::Wrap;
+
+type FromT = profiling_prelude_perf_types::event::EventScope;
+type IntoT = perf_event_rs::event::EventScope;
+
+impl From<&FromT> for Wrap<IntoT> {
+    fn from(value: &FromT) -> Self {
+        #[rustfmt::skip]
+        let val = match value {
+            FromT::User            => IntoT::User,
+            FromT::Kernel          => IntoT::Kernel,
+            FromT::Hv              => IntoT::Hv,
+            FromT::Idle            => IntoT::Idle,
+            FromT::Host            => IntoT::Host,
+            FromT::Guest           => IntoT::Guest,
+            FromT::CallchainKernel => IntoT::CallchainKernel,
```

Comment:
Abstract `CallchainKernel` and `CallchainUser` to a separate data struct, they don't belong to event scope.